dashboard/src/assets/fonts/Open_Sans/README.txt

```diff
@@ -0,0 +1,100 @@
+Open Sans Variable Font
+=======================
+
+This download contains Open Sans as both variable fonts and static fonts.
```

Comment:
Do we really have to include all of these fonts in the source code? Can't we use some NPM package for this? Does [`@fontsource/open-sans`](https://www.npmjs.com/package/@fontsource/open-sans) work?

Reply:
replaced